src/Features/CSharp/Portable/UseAnotherLiteral/CSharpUseAnotherLiteralCodeFixProvider.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseAnotherLiteral
```

Comment:
in terms of naming, this would likely be better as "ReplacDefaultWithLiteral"

Reply:
But a default literal *is* a literal.

Reply:
I think that's being too pedantic :)

Reply:
I think it would be nice for this to have a similar name as the other PR (which is named UseDefaultExpression). If this were to be named "Replace", I'd like to rename the other one as well. But I still think "use" is fine, unless you are totally opposed to it.

Reply:
"use another literal" simply feels too vague.  I have no idea from that what this is doing.  And, as this is actually an extremely specific sort of fix, it really doesn't convey that at all.

> The compiler makes this clear in every error message

But this isn't about the compiler's error message.  This is about the organization of features in the IDE codebase.  I'd really like to have a decent sense of understanding what a specific feature does from its name :)

> I think it would sound confusing to suggest it's not a literal.

There's no suggestion.  it's just describing what's going on in fairly human terms to anyone reading "this replaces usages of 'default' with a literal value instead".  that would be clear enough, and more clear than saying saying "this uses another literal".  

Note: i'm not saying it has to be ReplacDefaultWithLiteral.  That was just a suggestion.  The major point though is that the name shoudl be clearer about what this is actually doing.

Reply:
I will change the name if the design review decides to take this feature, but right now it doesn't look like it will have any future.

Reply:
> but right now it doesn't look like it will have any future.

Why do you say that?

Reply:
Sam said he is against it (and you did too).

Reply:
I'll hold off until the design is reviewed, thanks.

Reply:
I renamed it to ReplaceDefaultLiteral, since it is now a single code fix that replaces the default literal with either a default expression or another literal. I find ReplaceDefaultWithAnotherLiteralOrDefaultExpression rather long.